rules/windows/execution_register_server_program_connecting_to_the_internet.toml

```
@@ -2,12 +2,12 @@
 creation_date = "2020/02/18"
 ecs_version = ["1.6.0"]
 maturity = "production"
-updated_date = "2020/08/03"
+updated_date = "2020/08/12"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies the native Windows tools regsvr32.exe and regsvr64.exe making a network connection. This may be indicative of
+Identifies the native Windows tools regsvr32.exe, regsvr64.exe, RegSvcs.exe or RegAsm.exe making a network connection. This may be indicative of
```

```suggestion
Identifies the native Windows tools regsvr32.exe, regsvr64.exe, RegSvcs.exe, or RegAsm.exe making a network connection. This may be indicative of
```

For consistency with oxford comma. This needs to be re-linted too